packages/kbn-securitysolution-alerts-grouping/src/components/types.ts

```diff
@@ -1,13 +1,17 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
  */
 
-import type { GenericBuckets } from '../../../../common/search_strategy/common';
-
-export const DEFAULT_GROUPING_QUERY_ID = 'defaultGroupingQuery';
+// copied from common/search_strategy/common
+export interface GenericBuckets {
+  key: string | string[];
+  key_as_string?: string; // contains, for example, formatted dates
+  doc_count: number;
+}
```

Comment:
Include to GenericBucket this part:
```suggestion
groupByMultipleFields1?: {
    buckets?: GenericBuckets[];
    doc_count_error_upper_bound?: number;
    sum_other_doc_count?: number;
  };
 }
```

Reply:
I do not see the `groupByMultipleFields1` field in the code currently. can we wait to introduce it before committing the type?